tests-invoke

```diff
@@ -177,13 +177,6 @@ class PullTask(object):
         status["link"] = "log.html"
         status["extras"] = ["https://raw.githubusercontent.com/cockpit-project/bots/master/task/log.html"]
 
-        # Include information about which base we're testing against
-        if self.base:
-            subprocess.check_call(["git", "fetch", "origin", self.base])
-            commit = subprocess.check_output(["git", "rev-parse", "origin/" + self.base],
-                                             universal_newlines=True).strip()
-            status["base"] = commit
```

Comment:
Ack, seems the [sink](https://github.com/cockpit-project/cockpituous/blob/master/sink/sink) doesn't look at that field at all.